src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.tsx

```diff
@@ -0,0 +1,171 @@
+import { useDebouncedValue } from 'src/hooks/useDebounce';
+import useGetAppSettings from 'src/hooks/useGetAppSettings';
+import { useLocale } from 'src/hooks/useLocale';
+import { currencyFormat, dateFormatShort } from 'src/lib/intlFormat';
+import { formatNumber } from '../AccountSummary/AccountSummary';
+import {
+  FinancialAccountContext,
+  FinancialAccountType,
+} from '../Context/FinancialAccountsContext';
+import { FinancialAccountHeader } from '../Header/Header';
+import {
+  AccountTransactionTable,
+  FinancialAccountEntryTypeEnum,
+} from './AccountTransactionTable/AccountTransactionTable';
+import { useFinancialAccountEntriesQuery } from './financialAccountTransactions.generated';
+
+const Container = styled(Box)(() => ({
+  height: `calc(100vh - ${headerHeight})`,
+  overflowY: 'auto',
+}));
+
+const formatDateRange = (startDate?: DateTime, endDate?: DateTime) => {
+  if (!startDate) {
+    startDate = DateTime.local().minus({ months: 1 }).plus({ days: 1 });
+  }
+  if (!endDate) {
+    endDate = DateTime.local();
+  }
+  return `${startDate.toISODate()}..${endDate.toISODate()}`;
+};
+
+const defaultDateRange = formatDateRange();
+const defaultStartDate = defaultDateRange.split('..')[0];
+const defaultEndDate = defaultDateRange.split('..')[1];
+
+export const AccountTransactions: React.FC = () => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+  const {
+    accountListId,
+    financialAccountId,
+    activeFilters,
+    setActiveFilters,
+    hasActiveFilters,
+    searchTerm,
+  } = useContext(FinancialAccountContext) as FinancialAccountType;
+  const { appName } = useGetAppSettings();
+
+  useEffect(() => {
+    if (!hasActiveFilters) {
+      setActiveFilters({
+        dateRange: {
+          min: defaultStartDate,
+          max: defaultEndDate,
+        },
+      });
+    }
+  }, [hasActiveFilters]);
+
+  const dateRange = useMemo(() => {
+    if (!activeFilters?.dateRange?.min || !activeFilters?.dateRange?.max) {
+      return defaultDateRange;
+    }
+    return formatDateRange(
+      DateTime.fromISO(activeFilters.dateRange.min),
+      DateTime.fromISO(activeFilters.dateRange.max),
+    );
+  }, [activeFilters]);
+  const categoryId = activeFilters?.categoryId ?? '';
+  const wildcardSearch = useDebouncedValue(searchTerm, 500);
+
+  const { data } = useFinancialAccountEntriesQuery({
+    variables: {
+      input: {
+        accountListId,
+        financialAccountId: financialAccountId ?? '',
+        dateRange,
+        categoryId,
+        wildcardSearch,
+      },
+    },
+  });
+
+  const handleExportCSV = () => {
+    if (!data) {
+      // Alert user there is no data
+      // TODO: Add alert
+      return;
+    }
+    const columnHeaders = [
+      t('Date'),
+      t('Payee'),
+      t('Memo'),
+      t('Outflow'),
+      t('Inflow'),
+    ];
+    const convertDataToArray = data.financialAccountEntries.entries.reduce(
```

Comment:
Can this just be a `.map` instead of a reduce since we're creating one row per entry?